### DISCUSSION TITLE
SCIM endpoint for azure role provisioning

### DISCUSSION THREAD

**Discussion by onexbash** [outline/outline#7260] [Ideas]
This requests is coming from the Cratos GmbH: 
It would be super helpful to have a SCIM endpoint that can be used for provisioning of roles from Microsoft Entra. 
For example: 
Entra Group foo_bar has write access for the Outline collection bar_foo.

**Comment by acul021** [comment]
This should be doable today with https://github.com/acul021/scim-outline-adapter - a sidecar that exposes SCIM 2.0 and maps it to Outline's API. Entra groups become Outline groups (usable for collection permissions), and the Outline team role (admin/member/viewer) is derived from group membership via a configurable mapping. I only tested it against authentik's SCIM provider so far, so feedback from an Entra setup would be welcome. Details: https://github.com/outline/outline/discussions/12984
